pkgs/applications/audio/geonkick/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitLab, cmake, pkgconfig, redkite, libsndfile, rapidjson, libjack2, lv2, libX11, cairo }:
+
+stdenv.mkDerivation rec {
+  pname = "geonkick";
+  version = "2.2.0";
+
+  src = fetchFromGitLab {
+    owner = "iurie-sw";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0nb0q4pysgxpsm0zn5xgmb9xg2573pl8vx3b7q8pn8hr4pxrh8y2";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig ];
```

Comment:
In new expressions, `pkg-config` should be used instead of the `pkgconfig` alias.

Reply:
I keep hitting this one, I'll change it.  

When I made the package I searched for both options, found that pkgconfig is used way more, so I figured it was the correct one.

I guess nobody is willing to comb trough 7635 instances of ``pkgconfig`` and replace most, but not all, right?  Or is there another reason it is used more?

Reply:
Yeah, that is the case. Long long time ago nixpkgs did not allow dashes in attribute names so we omitted it. These days we are trying to move closer to upstream.